chsdi/templates/htmlpopup/bafu_waldreservate.mako

```diff
@@ -1,23 +1,22 @@
 <%inherit file="base.mako"/>
 
 <%def name="table_body(c, lang)">
-    <tr><td class="cell-left">${_('ch.bafu.waldreservate.objnummer')}</td>                    <td>${c['attributes']['objnummer'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.bafu.waldreservate.name')}</td>                         <td>${c['attributes']['name'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.bafu.waldreservate.gisflaeche')}</td>                   <td>${round(c['attributes']['gisflaeche'],2) or '-'}</td></tr>
+
+    <%
+       lang = 'de' if lang in ('de', 'rm') else lang
+       class_text = 'mcpfe_class_%s' % lang
+    %>
+
+    <tr><td class="cell-left">${_('ch.bafu.waldreservate.objnummer')}</td><td>${c['attributes']['objnummer'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.bafu.waldreservate.name')}</td><td>${c['attributes']['name'] or '-'}</td></tr>
```

Comment:
I'm not a specialist in Rumantsch Grischun - but Name is the only declaration being translated - and that is not consistent. This time Name is translated to Num